test/fixtures/basic/unmanaged-pod.yml.erb

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: unmanaged-pod-<%= deployment_id %>
+  labels:
+    type: unmanaged-pod
+    name: unmanaged-pod-<%= deployment_id %>
+    app: basic
+spec:
+  activeDeadlineSeconds: 20
+  restartPolicy: Never
+  containers:
+    - name: basic
+      image: hello-world:latest
+      imagePullPolicy: IfNotPresent
+      env:
+      - name: CONFIG
+        valueFrom:
+          configMapKeyRef:
+            name: configmap-data
+            key: datapoint2
```

Comment:
Yeah, it's so this can be used to test the case where you roll out a pod depending on a configmap that you forgot to include.